src/sagas/ReduxSaga.js

```diff
@@ -13,13 +13,14 @@ export default class ReduxSaga {
    *
    * @param {Generator} Saga implementation
    * @param {any} initial model
+   * @param {Function} dispatch function
    */
-  constructor(saga, model) {
+  constructor(saga, model, dispatch) {
     this.updateModel(model);
 
     this.dispatchSubject = new Subject();
     this.subscribeSubject = new Subject();
-    this.subscription = null;
+    this.subscription = this.subscribeSubject.subscribe(dispatch);
```

Comment:
Here's the trick, we just replicate behaviour of `subscribe` function, which now can be removed.